spotifycli/spotifycli.py

```diff
@@ -98,106 +129,190 @@ def get_arguments():
         ("--playpause", "plays or pauses the song (toggles a state)"),
         ("--lyrics", "shows the lyrics for the song"),
         ("--next", "plays the next song"),
-        ("--prev", "plays the previous song")
+        ("--prev", "plays the previous song"),
+        ("--export", "export lyrics"),
     ]
 
 
+def set_genius_token(token):
+    os.environ[TOKEN_ENV_VAR] = token
+
+    with open(os.path.join(expanduser("~"), TOKEN_FILE_NAME), "w") as f:
+        f.write(token)
+
+    print("Token was set")
+
+
+def load_genius_token():
+    file_path = os.path.join(expanduser("~"), TOKEN_FILE_NAME)
+
+    if os.path.isfile(file_path):
+        with open(os.path.join(expanduser("~"), TOKEN_FILE_NAME), "r") as f:
+            os.environ[TOKEN_ENV_VAR] = f.read()
+    else:
+        print(
+            "No genius api token was set. Some lyrics will not be found. "
+            "Use --token option to set it!"
+        )
+
+
 def show_version():
     print("1.6.0")
 
 
 def get_song():
     metadata = get_spotify_property("Metadata")
-    artist = metadata['xesam:artist'][0]
-    title = metadata['xesam:title']
+    artist = metadata["xesam:artist"][0]
+    title = metadata["xesam:title"]
     return (artist, title)
 
 
 def show_status():
     artist, title = get_song()
-    print(f'{artist} - {title}')
+    print(f"{artist} - {title}")
 
 
 def show_status_short():
     artist, title = get_song()
-    artist = artist[:15] + (artist[15:] and '...')
-    title = title[:10] + (title[10:] and '...')
-    print(f'{artist} - {title}')
+    artist = artist[:15] + (artist[15:] and "...")
+    title = title[:10] + (title[10:] and "...")
+    print(f"{artist} - {title}")
 
 
 def show_song():
     _, title = get_song()
-    print(f'{title}')
+    print(f"{title}")
 
 
 def show_song_short():
     _, title = get_song()
-    title = title[:10] + (title[10:] and '...')
-    print(f'{title}')
+    title = title[:10] + (title[10:] and "...")
+    print(f"{title}")
 
 
-def show_lyrics():
+# Normalize unicode string for a better possibility
+# of finding records in azlyrics
+def normalize(string):
```

Comment:
Are these comments required?
If there is need to comment function, then there's something wrong with its name.